airbyte-integrations/connectors/source-intercom/source_intercom/schemas/contact_segments.json

```diff
@@ -0,0 +1,29 @@
+{
+    "type": "object",
+    "properties": {
+        "type": {
+            "type": ["null", "string"]
+        },
+        "id": {
```

Comment:
is this the segment id? 

We will need that in order to map segments to contacts